sklearn/feature_extraction/text.py

```diff
@@ -1474,13 +1477,26 @@ class TfidfVectorizer(CountVectorizer):
 
     See also
     --------
-    CountVectorizer
-        Tokenize the documents and count the occurrences of token and return
-        them as a sparse matrix
-
-    TfidfTransformer
-        Apply Term Frequency Inverse Document Frequency normalization to a
-        sparse matrix of occurrence counts.
+    CountVectorizer : Produces a sparse representation of the counts using
+        scipy.sparse.csr_matrix.
+
+    TfidfTransformer : Converts the count matrix from CountVectorizer to a
+        normalized tf-idf representation. Tf is term frequency, and idf is
+        inverse document frequency. This is a common way to calculate the
+        count of a word relative to the appearance of a document.
+
+        The formula that is used to compute the tf-idf of term t is​
+        :math:`tf-idf(d, t) = tf(t) * idf(d, t)` and the idf is computed
+        as​ `idf(d, t) = log(\frac{n}{df(d, t)} + 1)`
```

Comment:
should this also have `:math:`?

Reply:
I can add `:math:` back in.  It's that I the html does not render wherever there is `\log` or `\frac` and I'm not sure what to do about that.